### DISCUSSION TITLE
Zoom and pan controls

### DISCUSSION THREAD

**Discussion by stuwilmur** [neocarto/bertin#121] [Ideas]
It would be great if bertin.js could include zoom and pan support, like [geooverview](https://github.com/neocarto/geoverview). Currently, I approximate this behaviour by controlling the `extent` parameter using some controls outside the map. Or, is this already possible somehow and I just haven't found the feature?

🙏 for Bertin ♥!

**Comment by neocarto** [comment]
Not sure to include that in bertin. Actually, i'm working on another library called geoviz with zoom and pan enable. Work in progress.

**Comment by stuwilmur** [comment]
No problem; I found geoviz previously, and wondered if this might include such a feature in future, so I look forward to it being developed.

**Reply by neocarto** [reply]
New release soon
